pkg/component/etcd/statefulset/statefulset.go

```diff
@@ -118,16 +122,56 @@ func (c *component) Wait(ctx context.Context) error {
 	return err
 }
 
-func (c *component) waitDeploy(ctx context.Context, sts *appsv1.StatefulSet, replicas int32) error {
-	return gardenerretry.UntilTimeout(ctx, defaultInterval, defaultTimeout, func(ctx context.Context) (bool, error) {
-		if err := c.client.Get(ctx, client.ObjectKeyFromObject(sts), sts); err != nil {
+func (c *component) getMostRecentPodCreationTimestamp(ctx context.Context, sts *appsv1.StatefulSet) (time.Time, error) {
```

Comment:
Can we rename the function as `getLatestPodCreationTime` ?

Reply:
done